x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -2473,3 +2473,13 @@ c2:l |c2_f:l |m2:i |m2_f:i |c:l
 1    |0      |2    |2      |19
 1    |1      |5    |5      |21
 ;
+
+
+statsByConstantExpression
+required_capability: fix_stats_by_foldable_expression
+FROM employees | eval x = [1,2,3], y = 5 + 6 | stats by y+1
```

Comment:
nit: we could also add the example with the explicit alias in the group `STATS BY yy = y+1`. And test cases with a non-empty list of aggs.